blazor/rich-text-editor/form-validation.md

```diff
@@ -13,64 +13,19 @@ This following sample demonstrate how to get the Rich Text Editor validation err
 
 ## Render the editor in a form
 
-```cshtml
-
-@using Syncfusion.Blazor.RichTextEditor
-@using System.ComponentModel.DataAnnotations;
-
-<div class="control-section">
-    <div class="content-container">
-        <div id="content" class="box-form" style="margin: 0 auto; max-width:750px; padding:25px">
-            <EditForm Model="@Model">
-                <DataAnnotationsValidator />
-                <SfRichTextEditor ShowCharCount="true" MaxLength="100" Placeholder="Type something..." @bind-Value="@Model.Description" />
-                <ValidationMessage For="@(() => Model.Description)" />
-                <div class="btn-grp">
-                    <button class="samplebtn e-control e-btn" type="reset" data-ripple="true">Reset</button>
-                    <button class="samplebtn e-control e-btn" type="submit" data-ripple="true">Submit</button>
-                </div>
-            </EditForm>
-        </div>
-    </div>
-</div>
-
-<style>
-    .box-form {
-        webkit-box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
-        box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
-    }
-
-    .btn-grp {
-        text-align: center;
-        margin-top: 15px;
-    }
-
-    .validation-message {
-        color: #f44336;
-        font-family: "Roboto", "Segoe UI", "GeezaPro", "DejaVu Serif", "sans-serif", "-apple-system", "BlinkMacSystemFont";
-        font-size: 12px;
-        font-weight: normal;
-    }
-</style>
-
-@code{
-    private class FormModel
-    {
-        [Required]
-        [MinLength(20, ErrorMessage = "Please enter at least 20 characters.")]
-        public string Description { get; set; }
-    }
-
-    private FormModel Model = new FormModel();
-}
-
-```
+{% tabs %}
+{% highlight razor tabtitle="~/edit-form.razor" %}
+
+{% include_relative code-snippet/edit-form.razor %}
+
+{% endhighlight %}
+{% endtabs %}
 
 ![Form Validation in Blazor RichTextEditor](./images/blazor-richtexteditor-form-validation.png)
 
 ## Validation Rules
 
-The Rich Text Editor is a textarea control. The Rich Text Editor also provides the functionality of character count and its validation. So, you can validate the Rich Text Editor's value on form submission by applying Validation Rules and Validation Message to the Rich Text Editor.
+The Rich Text Editor is a textarea control. The editor also provides the functionality of character count and its validation. So, you can validate the editor's value on form submission by applying Validation Rules and Validation Message to the editor.
```

Comment:
use this content - The Rich Text Editor provides the functionality of character count and its validation. So, you can validate the editor's value on form submission by applying validation attributes and validation messages to the editor.